assert/assertion_forward.go

```diff
@@ -3,6 +3,7 @@
 package assert
 
 import (
+	jsonmatch "github.com/stretchr/testify/assert/jsonmatch"
```

Comment:
Fix the import order please

Reply:
I think this is what is meant? https://github.com/stretchr/testify/pull/1690/commits/bdff01450338f98f2c043d0c08dda4de1ddc9fda